### ISSUE TITLE
[Feature request] [Build] Are you sure? Y/N

### ISSUE THREAD

**Issue by Thunderofnl** [modmore/Gitify#95]
I would like to see a "Are you sure? y/N" option before doing either build or build --force. I can foresee people accidentally typing build instead of extract on a production environment and then the website will lose some hours of progress or in some cases even days.

**Comment by Mark-H** [comment]
Did that actually happen? :fearful: 

I can imagine a feature like this becoming really annoying really quickly when developing, so not 100% sure about what the best way to do this would be. Maybe, tied in with the idea of a .gitifyrc (#48) this could be hidden behind an environment specific flag?

**Comment by Thunderofnl** [comment]
Yes it did... Someone (Won't say any names) went two arrow keys up instead of one... His last command was extract but the one before that was build (Glad it was without --force though). 

But back on topic. Yeah I think I could add that to the .gitifyrc issue. I first need to find time to develop that though, quite busy lately and I have no clue when the busyness stops...
